### ISSUE TITLE
Republish NPM package to reflect latest changes to package.json

### ISSUE THREAD

**Issue by RobbyCowell** [liip/TheA11yMachine#127]
Could someone republish the NPM package to reflect this change? https://github.com/liip/TheA11yMachine/commit/fb6acb811018ba94af9e76e81c6498e606bd25a1

It breaks `npm install` for me and likely anyone else who has this package in their dependencies.

**Comment by ldavidson** [comment]
Yes having same issue can npm install all the way up to 0.8.1 but after that get:

```
npm ERR! code ENOPACKAGEJSON
npm ERR! package.json Non-registry package missing package.json: HTML_CodeSniffer@git+https://github.com/squizlabs/HTML_CodeSniffer.git#master.
npm ERR! package.json npm can't find a package.json file in your current directory.
```

**Comment by RobbyCowell** [comment]
@ldavidson as a workaround, I went into the package.json for the TheA11yMachine package, and manually entered the change shown in the commit I linked above:

    - "HTML_CodeSniffer": "https://github.com/squizlabs/HTML_CodeSniffer#master",
    + "html_codesniffer": "^2.2.0",

Far from a perfect solution, but it should let you run npm install, at least it did for me.

Hope that helps!

**Comment by brendano86** [comment]
Ran into the same issue today trying to run `npm install -g the-a11y-machine`

**Comment by Hywan** [comment]
cc @jeanmonod
